.changes/run-return-mobile.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri-runtime": minor:feat
+---
+
+`Runtime::run_return` now must also be implemented on mobile targets.
```

Comment:
I guess this can be omitted because the function is only added in this PR (see other changelog entry?)